blog/20140130-mate-desktop-live-cd.de.md

```diff
@@ -70,22 +70,22 @@ aber wir haben einige zusätzliche nicht zum MATE-Desktop gehörenden Anwendunge
 um die Live-CD ein wenig nützlicher und unterhaltsamer zu machen.
 
   * [Firefox](https://www.mozilla.org/firefox/)   - Stand-Alone Web-Browser von mozilla.org
-  * [GParted](http://gparted.sourceforge.net)   - Ein Klon von Partition Magic, Front-End für GNU Parted
-  * [HardInfo](http://hardinfo.berlios.de/wiki/index.php/Main_Page)  - Ein System-Informations- und Benchmark-Tool
-  * [Hexchat](http://hexchat.github.io/)   - IRC-Client, konfiguriert für die automatischen Verbindung zu [#mate@freenode](https://webchat.freenode.net/?channels=#mate)
-  * [Onboard](http://www.gnome.org/projects/orca)      - Screen-Reader für blinde bzw. sehbeinträchtigte Personen
-  * [Orca](http://www.gnome.org/projects/orca)      - On-Screen-Keyboard, nützlich für motorisch eingeschränkte Personen
-  * [Pidgin](http://pidgin.im/)    - Multi-Protokoll Instant-Messaging Client
-  * [Truecrypt](http://www.truecrypt.org/) - Freie, quelloffene, cross-plattform Festplatten-Verschlüsselungs-Software
-  * [Xnoise](http://www.xnoise-media-player.com/)    - Medien-Player mit einer schicken GUI, hoher Geschwindigkeit und vielen Features
+  * [GParted](https://gparted.sourceforge.net)   - Ein Klon von Partition Magic, Front-End für GNU Parted
+  * [HardInfo](https://hardinfo.berlios.de/wiki/index.php/Main_Page)  - Ein System-Informations- und Benchmark-Tool
+  * [Hexchat](https://hexchat.github.io/)   - IRC-Client, konfiguriert für die automatischen Verbindung zu [#mate@freenode](https://webchat.freenode.net/?channels=#mate)
+  * [Onboard](https://www.gnome.org/projects/orca)      - Screen-Reader für blinde bzw. sehbeinträchtigte Personen
+  * [Orca](https://www.gnome.org/projects/orca)      - On-Screen-Keyboard, nützlich für motorisch eingeschränkte Personen
+  * [Pidgin](https://pidgin.im/)    - Multi-Protokoll Instant-Messaging Client
+  * [Truecrypt](https://www.truecrypt.org/) - Freie, quelloffene, cross-plattform Festplatten-Verschlüsselungs-Software
+  * [Xnoise](https://www.xnoise-media-player.com/)    - Medien-Player mit einer schicken GUI, hoher Geschwindigkeit und vielen Features
 
 ## Creative-Commons-Inhalte
 
 Wir haben die folgenden Inhalte, lizensiert unter Creative-Commons, hinzugepackt:
 
-  * [Think Python](http://www.greenteapress.com/thinkpython/)                        - Zum Lesen mit Atril
-  * [Nine in Nails Ghosts I - IV](http://ghosts.nin.com/main/home) Ein Album - Zum Hören mit Xnoise
-  * [Nine in Nails Ghosts I - IV](http://ghosts.nin.com/main/home) Photos  - Zum Betrachten mit "Eye of MATE"
+  * [Think Python](https://www.greenteapress.com/thinkpython/)                        - Zum Lesen mit Atril
+  * [Nine in Nails Ghosts I - IV](https://ghosts.nin.com/main/home) Ein Album - Zum Hören mit Xnoise
+  * [Nine in Nails Ghosts I - IV](https://ghosts.nin.com/main/home) Photos  - Zum Betrachten mit "Eye of MATE"
```

Comment:
http://ghosts.nin.com/main/home and https://ghosts.nin.com/main/home don't work

Reply:
Dropped

Reply:
Btw. most of all this blog entries i don't find on website.